src/Tasks/Microsoft.NET.Build.Extensions.Tasks/GetAssemblyInfo.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Framework;
+using Microsoft.NET.Build.Tasks.ConflictResolution;
```

Comment:
This file should not reference ConflictResolution, I should move FileUtilities.GetFileVersion to a "higher" name space right?